dvc/remote/local/__init__.py

```diff
@@ -313,13 +315,13 @@ def status(
         show_checksums=False,
         download=False,
     ):
-        logger.info(
+        logger.debug(
             "Preparing to collect status from {}".format(remote.path_info)
```

Comment:
Though, maybe showing where we are collecting status from would be actually nice. What do you think @casperdcl ? Could we make it look good somehow? Or should we leave it out?

Reply:
idk. if we want to replicate the verbosity of `git`:

```
Fetching origin
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (27/27), done.
remote: Compressing objects: 100% (13/13), done.
remote: Total 27 (delta 16), reused 25 (delta 14), pack-reused 0
Unpacking objects: 100% (27/27), done.
From github.com:casperdcl/hidden
   947f0f5..f082d34  master    -> origin/master
```

then maybe we really do want:

```
Fetching status: https://remote.dvc.org/dvc
100%|██████████|Querying local cache           3/3 [00:00<00:00,  6.02kmd5/s]
100%|██████████|Querying remote cache          3/3 [00:00<00:00,   3.97md5/s]
100%|██████████|Analysing status               3/3 [00:00<00:00, 5.46kfile/s]
100%|██████████|Checkout                       3/3 [00:00<00:00,   791file/s]
```

Reply:
Maybe git is not the best example in this case, especially that our operations take more time, since we are working with much bigger amounts of data. We could include the name of the remote into `cache_exists` bar for the remote though. Currently, we are printing md5s, which is not that informative and not user-friendly. If we could have progress bars (that are gone when they are done, just like right now) with a name of something like `Querying cache in .dvc/cache` and `Querying cache in s3://mybucket/mypath`, it would probably be more friendly. Also, in the asciinema snippet that you've shared above, you can see that only `Checkout` progress bar is now left behind, which seems weird. Same as with status progress bars, it is only useful when it is doing something, so maybe let's clear it as well.

Reply:
@efiop ok, it would be helpful if you could say what final output should be. Completely blank?

Reply:
In this case, i think so, it is going to be blank for now :(